onchain/rollups/test/foundry/dapp/helper/update-proofs.sh

```diff
@@ -89,51 +68,13 @@ then
     docker build -t cartesi/server-manager-gen-proofs:devel .
     echo
 
-    echo3 $GREEN "Installing Python packages..."
-    echo
-
-    # Install Python packages with pip3
-    pip3 install -r requirements.txt
-    echo
-
-    echo3 $GREEN "Generating dummy inputs..."
-    echo
-
-    # Generate a dummy inputs JSON
-    inputs="[]"
-    sender="0xdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef"
-    payload="0x"
-    for (( i=0; i<$NINPUTS; i++ ))
-    do
-        inputs=`echo "$inputs" | jq ".[$i].sender = \"$sender\""`
-        inputs=`echo "$inputs" | jq ".[$i].payload = \"$payload\""`
-    done
-    echo "$inputs" > inputs.json
-
-    echo3 $GREEN "Generating script to be run on docker image..."
-    echo
-
-    genscript
-    echo
-
     # Give execute permission to script
     chmod +x gen-proofs.sh
 
     echo3 $GREEN "Running docker image to generate epoch status..."
     echo
 
-    runmachine
-    echo
-
-    echo3 $GREEN "Processing and formatting epoch status..."
-    echo
-
-    b64to16
-
-    echo3 $GREEN "Generating Solidity library with proofs..."
-    echo
-
-    genlib
+    runmachine --setup
```

Comment:
instead of `runmachine --setup`, can we simply do `echo "[]" > output/inputs.json` here. And `runmachine` doesn't have to care about `--setup` or not?

Reply:
My intention was for the `update-proofs.sh` to abstract the internals of `gen-proofs.sh`. Besides, this would disrupt our logical separation of `input` (only read by the docker image) and `output` (only written by the docker image).

Reply:
Then can we remove this line `runmachine --setup` as we don't need to run the server-manager (there's no inputs to process).
And in `gen-proofs.sh`, we can simplify the codes [here](https://github.com/cartesi/rollups/blob/b3c0bd0293019a3cb299d3144ebfeca61873fe31/onchain/rollups/test/foundry/dapp/helper/gen-proofs.sh#L121):
from
```
if [ $# -ge 1 ] && [ $1 == "--setup" ]
then
    # Then create an empty inputs.json
    echo "[]" > output/inputs.json
else
    [ -f output/inputs.json ] ||
        (echo 'Hint: run `yarn proofs:setup`.' && echo && exit 1)

    # Otherwise, update inputs.json with forge test output
    filter=`awk -f jqFilter.awk input/forge_test_output.txt`
    updated_inputs=`jq "${filter}" output/inputs.json`
    echo "${updated_inputs}" > output/inputs.json
fi
```
to
```
if [ ! -f output/inputs.json ]
then
    # Then create an empty inputs.json
    echo "[]" > output/inputs.json
fi
# update inputs.json with forge test output
filter=`awk -f jqFilter.awk input/forge_test_output.txt`
updated_inputs=`jq "${filter}" output/inputs.json`
echo "${updated_inputs}" > output/inputs.json
```
This can improve readability and run time as we only run the `gen-proofs.sh` once rather than twice

Reply:
Hey, I've removed the `--setup` option from the `gen-proofs.sh` script in my latest commit. :-)